source/_integrations/mcp_server.markdown

```diff
@@ -111,10 +111,11 @@ to allow Claude for Desktop to access Home Assistant using the SSE transport.
 4. Add a new MCP server to the JSON file. You need to set the `SSE_URL` to the URL you use to
    connect to Home Assistant with the path `/mcp_server/sse`. You will also need to set `API_ACCESS_TOKEN`
    to the long live access token created above in the [access control instructions](#access-control)
+
     ```json
     {
       "mcpServers": {
-        "Home Assistant": {
+        "home-assistant": {
```

Comment:
Is this required or a style preference? not sure this needs to chnange necessarily